methods/difference/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+* Returns a new array that is a copy of the receiver,
+* removing any items that also appear in any of the arrays
+* given as arguments.
+* The order is preserved from the original array.
+*
+* @param {Array} array
+* @param {...*} otherArrays
+*
+* @return {Array} Returns the diff array
+*
+* @example
+* difference([1, 1, 2, 2, 3, 3, 4, 5], [1, 2, 4]); // => [ 3, 3, 5 ]
+* difference([1, 'c', 'yep'], [1], [ 'a', 'c' ]);  // => ["yep"]
+ */
+export default function difference(array, ...otherArrays) {
+    let resultArray = array.slice(0);
```

Comment:
@aldnav I suppose the `.slice()` call here is to create a new copy of the array object to avoid modifying it? If that's the case, I don't think it's already necessary since `.filter()` already creates a new array, so using the `array` variable directly will not modify the original array.

Reply:
@aldnav You can also make it fancy by using `.reduce()` 😆